### PR TITLE
Adds back the JS Convection API

### PR DESCRIPTION
**Pull request by orta**
Paired with @sweir27 on bringing back the JS API for Convection, we've built it with the expectation of this being the API that we'll use when stitching. Now the ENV var switches between a working JS implementation, and a stitched implementation. They're not 100% fit _today_, but it's built with *client side* future proofing in mind.

Opted to having a `submission` always be called `consignment_submission` inside metaphysics. So, things that would need to be done in stitching:

* Rename mutations to something like `createConsignmentSubmission`
* Mutations in Convection change their return types to be `consignment_submission: [data]`
* `Submission` -> `ConsignmentSubmission`

### CODE REVIEW

File: schema/aggregations/aggregation_count.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // @ts-check
+import type { GraphQLFieldConfig } from "graphql"
```
Comment (alloy) [inline schema/aggregations/aggregation_count.js]: Is this Flow/TS mix intentional?

File: schema/me/consignments/submissions.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      type: GraphQLBoolean,
+      default: true,
+    },
+  },
+  description: "A list of the current user’s consignment submissions",
+  resolve: (root, options, _request, { rootValue: { submissionsLoader } }) => {
+    // TODO: This should get replaced in stitching,
+    //       it _does not_ handle pagination at API level, so you'll need a
+    //       list size that's a little higher than average while there
+    //       are still small amounts of data.
```
Comment (alloy) [inline schema/me/consignments/submissions.js]: I don’t understand what this comment is trying to convey. Is it important for drive-by readers to understand it or a note to self?

Comment (orta) [reply schema/me/consignments/submissions.js]: drive-by - basically, today you can use this but it will only paginate once and think it's done. In the future the stitched version will paginate correctly.

File: schema/aggregations/aggregation_count.js (hunk 1 above)
Comment (orta) [reply schema/aggregations/aggregation_count.js]: No, came with the revert - will remove
